src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ThrottlingRerouteSpec.groovy

```diff
@@ -192,8 +194,10 @@ class ThrottlingRerouteSpec extends HealthCheckSpecification {
         northbound.getAllFlows().empty
 
         and: "Related switches have no excess rules"
-        pathHelper.getInvolvedSwitches(PathHelper.convert(path)).each {
-            verifySwitchRules(it.dpId)
+        Wrappers.wait(RULES_DELETION_TIME) {
```

Comment:
Same question. It feels like flow should remain in progress until everything's removed. Isn't that expected?